dspace-api/src/main/resources/Messages.properties

```diff
@@ -1640,12 +88,6 @@ org.dspace.checker.SimpleReporterImpl.size
 org.dspace.checker.SimpleReporterImpl.source                                    = Source
 org.dspace.checker.SimpleReporterImpl.store-number                              = Store Number
 org.dspace.checker.SimpleReporterImpl.unchecked-bitstream-report                = The following is a UN-CHECKED BITSTREAM REPORT report for
-org.dspace.content.Collection.untitled                                          = Untitled
-org.dspace.content.Community.untitled                                           = Untitled
-org.dspace.eperson.LDAPAuthentication.title                                     = Enter LDAP Netid and Password
-org.dspace.eperson.PasswordAuthentication.title                                 = Enter DSpace Username and Password
-org.dspace.eperson.X509Authentication.title                                     = Enter DSpace using Web Certificate
```

Comment:
Referenced (commented) in config/modules/authentication-x509.cfg

Reply:
Good catch...I've restored this message in https://github.com/DSpace/DSpace/pull/3161/commits/a8d99bba15b4df491fda2390fe3611ef2e6336fd